exercism/bowling/bowling.js

```diff
@@ -0,0 +1,88 @@
+const Bowling = function(rolls) {
+  this.rolls = rolls
+
+}
+
+Bowling.prototype.score = function() {
+
+  let score = 0
+
+  if (this.rolls.length === 0) {
+    throw new Error('Score cannot be taken until the end of the game')
+  }
+
+  let tens = 0
+  for (let a = 0; a < this.rolls.length; a++) {
+
+    if (this.rolls[a] === 10) {
+      tens = tens + 1
+    }
+  }
+
+  if (this.rolls.length < 20 - tens || this.rolls[19 - tens] === 10 &&
```

Comment:
This check can probably be generalized a little more or be written as a predicate function for better-looking code. That big of a check looks messy and confusing to me.